cPP_DSC.adoc

```diff
@@ -672,11 +672,17 @@ a|[cols=".^1,3",options=noheader,grid="none",frame="none"]
 
 !FPT_STM.1 !Reliable Time Stamps
 
-!FCS_ENT_EXT.1 !Entropy for External IT Entities
+!FCS_RBG.6 !Random Bit Generation Service
+
+!FCS_RBG.2 !Random Bit Generation (External Seeding)
```

Comment:
I dropped all the SFRs in (I don't know if I put them in the right place off hand). They should be selection-based (or optional). I definitely didn't get all the details for those SFRs integrated into the cPP, so that would need to be cleaned up.

Reply:
If we can agree to this version, then we can start looking at what should go where as needed. Things can always be removed (or re-added), this is a rough filling in so we can discuss, it is not meant to be the final with no further changes.